src/vivarium_gates_nutrition_optimization_child/data/loader.py

```diff
@@ -405,6 +406,37 @@ def load_emr_from_csmr_and_prevalence(key: str, location: str) -> pd.DataFrame:
     return data
 
 
+def load_neonatal_deleted_csmr(key: str, location: str) -> pd.DataFrame:
+    """Get GBD 2019 CSMR data with 2021 age groups and zero out neonatal age groups."""
+    allowed_keys = [data_keys.DIARRHEA.CSMR, data_keys.LRI.CSMR, data_keys.MALARIA.CSMR]
+    if key not in allowed_keys:
+        raise ValueError(f"Unrecognized key {key}")
+
+    data = load_standard_gbd_2019_data_as_gbd_2021_data(key, location)
+    data.loc[data.reset_index()["age_start"] < metadata.NEONATAL_END_AGE, :] = 0
+    return data
+
+
+def load_post_neonatal_birth_prevalence(key: str, location: str) -> pd.DataFrame:
```

Comment:
If we're setting birth prevalence to the post-neonatal prevalence, we should be setting all transition rates in the neonatal age groups to zero as well. Confirm with Ali that this is correct.

Reply:
Confirmed that is what we want.